dist/index.html

```diff
@@ -4,9 +4,23 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <title>Rick and Morty App</title>
 <script defer src="main.js"></script></head>
 <body>
-  
+  <h1>TEST</h1>
+  <footer class="fixed-bottom text-center border-top pt-2">
+    <p>Created by</p>
+    <div class="d-flex justify-content-around">
+      <div>
+        <p>David Alvarez</p>
+        <p>GitHub: <a href="https://github.com/petumazo">petumazo</a></p>
+      </div>
+      <div>
+        <p>Alef Garrido</p>
+        <p>GitHub: <a href="https://github.com/alef-garrido">lef-garrido</a></p>
```

Comment:
Adding the reference to our github profiles is a nice detail. 
I'll just suggest to add a little bit of margin to the bottom since you added the bootstrap cdn already.  
Centered elements are looking good!

Reply:
Thanks for the feedback. The suggestion was followed and a bit of margin was added to the bottom.